### PR TITLE
Improve readability of bde_control tables

### PR DESCRIPTION
**Pull request by strk**
Description of some fields are still confusing to me,
but the PR can be improved with help from @palmerj.

In particular, it isn't clear to me the description of
row_tol_warning and row_tol_error (ratio of what?) and
there's a reference to a cbe_tables.tablekeycolumn column
which is not clear to me (where's that table?)

### CODE REVIEW

File: sql/01-bde_control_tables.sql [hunk 1]
```diff
@@ -92,21 +96,46 @@ GRANT SELECT ON TABLE upload_table TO bde_user;
...
+
+last_level0_dataset is the dataset id of the last level 0 uploaded.
+
 upl_id_lock is the id of an upload currently locking the table (this is
-  in the sense of a process lock, not a database lock)
-row_tol_warning is the minimum ratio of the number of number of rows in the table after a level 0 update  to the number before the update before a warning is logged
-row_tol_error is the minimum ratio of the number of number of rows in the table after a level 0 update  to the number before the update before an exception is thrown
+in the sense of a process lock, not a database lock).
+
+row_tol_warning is the minimum ratio of the number of number of rows
```
Comment (palmerj) [inline sql/01-bde_control_tables.sql]: Ah yes those words are pretty mixed up in terms of grammar and looking at the parameter definition and what it's doing it pretty confusing. What was I on when I wrote this? Definitely needed a peer review!

The parameter is a maximum percentage ratio between the current table row count and the new table row count after an incremental level_0 update before a warning/error is raised. So a value of 0.95 a would mean the less than a 5% row count change is allowed before a warning/error is raised. The reason for these parameters is due to past operational issues we had with Landonline dumping out partial BDE exports that only contained a small number of the table's total rows, but still had a valid headers.

File: sql/01-bde_control_tables.sql [hunk 2]
```diff
@@ -92,21 +96,46 @@ GRANT SELECT ON TABLE upload_table TO bde_user;
...
-last_upload_id is the id of the last upload job affecting this table
-last_upload_dataset is the dataset id of the last level 5 or 0 uploaded (since the level 0 upload will override any level 5 uploads that have been applied).
-last_upload_type is either 0 or 5
-last_upload_incremental is true if the table data was updated, and false if the table data was completely refreshed from a level 0
+Tracks the status of uploads for each table.
+
+key_column is the name of a unique non-composite, not null
+integer or bigint column used for identifying the table row for
+incremental updates. This identifier must be the same as defined in
+the cbe_tables.tablekeycolumn (?).
```
Comment (strk) [inline sql/01-bde_control_tables.sql]: Thanks, updated with bae26e2

File: sql/01-bde_control_tables.sql (hunk 1 above)
Comment (strk) [inline sql/01-bde_control_tables.sql]: Rephrased in 90e9f34

**Comment by strk** [comment]
Now that the text is ok I'll split the table and fields comments (wanted to get the descriptions right first)

**Comment by strk** [comment]
And with 8dc8fd1 we also have column-level comments. Ready to merge.

**Comment by strk** [comment]
@palmerj is it ok to merge this now ?

**Comment by imincik** [comment]
@strk , it is not mission critical PR. Please leave it for @palmerj once he will be able to respond.

**Comment by imincik** [comment]
@palmerj , thanks.

**Pull request merged**